cmd/server/pactasrv/conv_pacta_to_oapi.go

```diff
@@ -26,17 +26,32 @@ func initiativeToOAPI(i *pacta.Initiative) (*api.Initiative, error) {
 	}, nil
 }
 
+func userToOAPI(user *pacta.User) (*api.User, error) {
+	if user == nil {
+		return nil, errorInternal(fmt.Errorf("userToOAPI: can't convert nil pointer"))
```

Comment:
I hear your point about keeping all the `conv` stuff in the `pactasrv` package so you can access these, but my counterpoint would be that determining the 'type' of an error truly is a concern of the server, and this should just return a `fmt.Errorf(...)` or some `conv`-specific error that the `srv` layer then converts into an `errorWhatever`

Functionally, it probably doesn't matter, just cleaner abstractions and easier testing